src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/SocketsHttpHandler.cs

```diff
@@ -372,6 +372,21 @@ public bool EnableMultipleHttp2Connections
             }
         }
 
+        /// <summary>
+        /// Gets or sets a value that indicates whether additional HTTP/3 connections can be established to the same server
+        /// when the maximum of concurrent streams is reached on all existing connections.
+        /// </summary>
```

Comment:
should we document that this is outside [RFC](https://datatracker.ietf.org/doc/html/rfc9114#section-3.3-4)?

> Clients SHOULD NOT open more than one HTTP/3 connection to a given IP address and UDP port, where the IP address and port might be derived from a URI, a selected alternative service ([[ALTSVC](https://datatracker.ietf.org/doc/html/rfc7838)]), a configured proxy, or name resolution of any of these. A client MAY open multiple HTTP/3 connections to the same IP address and UDP port using different transport or TLS configurations but SHOULD avoid creating multiple connections with the same configuration.

Reply:
We should fix https://learn.microsoft.com/en-us/dotnet/api/system.net.http.socketshttphandler.enablemultiplehttp2connections?view=net-8.0 as well, as it's the same thing.

Reply:
> Gets or sets a value that indicates whether additional HTTP/2 connections can be established to the same server ~~when the maximum number of concurrent streams is reached on all existing connections~~.

While we're at it, we should also consider removing the comment about all existing connections, as we could decide to break that when making changes to the pool (e.g. #81529 / #66944).

Reply:
https://github.com/dotnet/dotnet-api-docs/pull/9921